src/documents/tests/test_barcodes.py

```diff
@@ -294,7 +294,7 @@ def test_scan_file_for_separating_barcodes(self):
         )
 
         self.assertEqual(doc_barcode_info.pdf_path, test_file)
-        self.assertListEqual(separator_page_numbers, [0])
+        self.assertDictEqual(separator_page_numbers, dict([(0, False)]))
```

Comment:
The conversion from a list of tuples to a dict here is extra work.  It would be more clear to just write the dictionary.